src/scss/_file-input.scss

```
@@ -0,0 +1,11 @@
+/// @access private
+/// @output Styling for file inputs
+@mixin _oFormsFileInput(
+) {
+
+	.o-forms-input--file ::file-selector-button {
+		@include oButtonsContent($opts: (
+			'type': 'primary'
+		));
```

it'd be great to put a little space between the button and copy of the input
```suggestion
		));
		margin-right: oSpacingByName('s1');
```

---

I was just looking at this 😂 I was searching the registry for the spacing component haha